#mission type definition
uint8 POINT=0
uint8 CLINE=1
uint8 CIRCLE=2

# mission type
int64 mission_type

# followed common line 
CommonLine[] lines

# followed circle
CommonCircle circle
